module_utils/nxos/facts/facts.py

```diff
@@ -0,0 +1,50 @@
+from ansible.module_utils.nxos.argspec.facts.facts import FactsArgs
+from ansible.module_utils.nxos.argspec.interfaces.interfaces import InterfaceArgs
+from ansible.module_utils.nxos.facts.base import FactsBase
+from ansible.module_utils.nxos.facts.interfaces.interfaces import InterfacesFacts
+
+
+class Facts(FactsArgs, FactsBase):
+
+    VALID_SUBSETS = [
+        'net_configuration_interfaces',
```

Comment:
Add this value in the argspec for `class FactsArgs(ArgspecBase)`
```
 'gather_subset': dict(default=['all'], choices=['net_configuration_interfaces', 'all'], type='list')
```

Reply:
+1